### ISSUE TITLE
UserId not found Exception

### ISSUE THREAD

**Issue by bibibilu** [GetStream/stream-chat-android#1578]
The crash stack trace:
```
java.lang.IllegalArgumentException: 
  at io.getstream.chat.android.livedata.repository.builder.RepositoryFacadeBuilder$build$getUser$1.invokeSuspend (RepositoryFacadeBuilder.kt:52)
  at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith (ContinuationImpl.kt:33)
  at kotlinx.coroutines.DispatchedTask.run (DispatchedTask.kt:106)
  at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely (CoroutineScheduler.kt:571)
  at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask (CoroutineScheduler.kt:750)
  at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker (CoroutineScheduler.kt:678)
  at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run (CoroutineScheduler.kt:665)
```

It seems that the issue begin to occur in version after and including 4.6.0. I have never been able to reproduce it in development but it caused so many crashes in production world. I have already used new ChatDomain constructor without user and used connectUser instead of setUser to listen for user changer in ChatClient. Do you have any ideas how to fix it? Thanks!

**Comment by JcMinarro** [comment]
Hi @bibibilu 
It was already fixed on #1523 
Could you check with the [last version](https://github.com/GetStream/stream-chat-android/releases/latest)

**Comment by bibibilu** [comment]
Yes, it occurs in both 4.6.0 and 4.7.0.
